Neos.ContentRepository.Core/Classes/Projection/ContentGraph/Node.php

```diff
@@ -127,29 +93,19 @@ private function __construct(
         public ?NodeName $name,
         public NodeTags $tags,
         public Timestamps $timestamps,
-        public VisibilityConstraints $visibilityConstraints,
-        ContentStreamId $contentStreamId
+        public VisibilityConstraints $visibilityConstraints
     ) {
         if ($this->classification->isTethered() && $this->name === null) {
             throw new \InvalidArgumentException('The NodeName must be set if the Node is tethered.', 1695118377);
         }
-        // legacy to be removed before Neos9
-        $this->nodeAggregateId = $this->aggregateId;
-        $this->nodeName = $this->name;
-        $this->subgraphIdentity = ContentSubgraphIdentity::create(
-            $contentRepositoryId,
-            $contentStreamId,
-            $dimensionSpacePoint,
-            $visibilityConstraints
-        );
     }
 
     /**
      * @internal The signature of this method can change in the future!
      */
-    public static function create(ContentRepositoryId $contentRepositoryId, WorkspaceName $workspaceName, DimensionSpacePoint $dimensionSpacePoint, NodeAggregateId $aggregateId, OriginDimensionSpacePoint $originDimensionSpacePoint, NodeAggregateClassification $classification, NodeTypeName $nodeTypeName, PropertyCollection $properties, ?NodeName $nodeName, NodeTags $tags, Timestamps $timestamps, VisibilityConstraints $visibilityConstraints, ContentStreamId $contentStreamId): self
+    public static function create(ContentRepositoryId $contentRepositoryId, WorkspaceName $workspaceName, DimensionSpacePoint $dimensionSpacePoint, NodeAggregateId $aggregateId, OriginDimensionSpacePoint $originDimensionSpacePoint, NodeAggregateClassification $classification, NodeTypeName $nodeTypeName, PropertyCollection $properties, ?NodeName $nodeName, NodeTags $tags, Timestamps $timestamps, VisibilityConstraints $visibilityConstraints): self
```

Comment:
```suggestion
    public static function create(ContentRepositoryId $contentRepositoryId, WorkspaceName $workspaceName, DimensionSpacePoint $dimensionSpacePoint, NodeAggregateId $aggregateId, OriginDimensionSpacePoint $originDimensionSpacePoint, NodeAggregateClassification $classification, NodeTypeName $nodeTypeName, PropertyCollection $properties, ?NodeName $name, NodeTags $tags, Timestamps $timestamps, VisibilityConstraints $visibilityConstraints): self
```

for consistency (while we're at it)

Reply:
adjusted :)